test/cmdlineTests/model_checker_timeout_all/args

```diff
@@ -1 +1 @@
---model-checker-engine all --model-checker-timeout 1000
+--model-checker-engine all --model-checker-timeout 1000 --model-checker-targets all
```

Comment:
Had to add this here because more properties are checked besides assertions. Same for the tests below.